src/prop.ts

```diff
@@ -197,18 +198,35 @@ function baseProp(rawOptions: any, Type: any, target: any, key: string, whatis:
     initAsObject(name, key);
   }
 
+  let refType;
+
+  switch (rawOptions.itemsRefType || rawOptions.refType) {
```

Comment:
change to lowercase to keep everything normal, like above, and everything following
```suggestion
  switch ((rawOptions.itemsRefType || rawOptions.refType).toLowerCase()) {
```